packages/checkout-ui-extensions/src/extension-points/api/standard/index.ts

```diff
@@ -287,6 +310,13 @@ export interface StandardApi<
    */
   customAttributes: StatefulRemoteSubscribable<Attribute[] | undefined>;
 
+  /**
+   * Customer account from the buyer. This value will update when there's a
+   * change in the account. The value is undefined if the buyer doesn't have
+   * any account.
+   */
+  customerAccount: StatefulRemoteSubscribable<CustomerAccount | undefined>;
```

Comment:
cc/ @sjanczy2 the changes in this file expose all the new APIs

Reply:
@sjanczy2 just a heads up that merging this PR won't release the packages. It will require one more step -- I'll ping you.